### PR TITLE
Misc maintenance

### PR DESCRIPTION
**Pull request by YesThatAllen**
~~\- Updated steps for getting started with a local version~~
- updated urls using https when possible
- minor spacing cleanup

### CODE REVIEW

**Comment by YesThatAllen** [comment]
Seems like the spacing changes [in this PR](https://github.com/2factorauth/twofactorauth/pull/2043#issuecomment-248461318) may go against this rubocop PR. Good thing I got it in before that went to master!

**Comment by Carlgo11** [comment]
Except I just merged it

**Comment by YesThatAllen** [comment]
Ha ha ha. Happy to rebase if the rest of the commits are desired.

**Comment by Carlgo11** [comment]
@YesThatAllen I made some changes to the clean-up to honour our .editorconfig rules.
I also removed 6cc5917b95111f969f6b257f03a1911d02566145 as I'm not as certain on how good that commit turned out.
Maybe commit that in a separate PR?

**Comment by Jawshy** [comment]
Oh noes. Looks like a recent merge has created a new merge conflict. @YesThatAllen, could you please resolve the conflict?

**Comment by YesThatAllen** [comment]
This all started, fwiw, when my text editor wouldn't save [this line in the pastebin entry](https://github.com/2factorauth/twofactorauth/blob/1c9c46005bed79a3f5ba28860b999102f35b54f3/_data/other.yml#L256) without forcibly removing that one trailing space. Not sure what was up with that.

File: _config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ author: Josh Davis
 description:
     List of sites with Two Factor Auth support which includes SMS, email,
     phone calls, hardware, and software.
-url: http://twofactorauth.org
+url: https://twofactorauth.org
```
Comment (Carlgo11) [inline _config.yml]: We have to use HTTP on this line as it otherwise won't let users with incompatible browsers access our site without HTTPS.

**Comment by YesThatAllen** [comment]
No worries, cleaned up the set of commits while I was at it.

**Comment by YesThatAllen** [comment]
> Not sure what was up with that.

Ah, til that my editor supports http://editorconfig.org/ out of the box.

File: _config.yml (hunk 1 above)
Comment (Jawshy) [reply _config.yml]: Should a comment be appended so that anyone else that stumbles upon this will know why HTTP - not HTTPS - is used?

**Pull request merged**